app/models/users/pubcookie_user.rb

```diff
@@ -12,7 +12,7 @@ def self.authenticate(uwnetid, password = nil, require_identity = nil, attach_me
     if u.nil?
       pr = PersonResource.find(uwnetid) rescue nil
       return false if pr.nil?
-      u = PubcookieUser.create :login => uwnetid
+      u = PubcookieUser.create :login => uwnetid, :provider => 'shibboleth', :uid => uwnetid
```

Comment:
Line is too long. [92/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.